assets/chezmoi.io/docs/user-guide/password-managers/azure-key-vault.md

```diff
@@ -0,0 +1,25 @@
+# Azure Key Vault
+
+chezmoi includes support for [Azure Key Vault secrets](https://learn.microsoft.com/en-us/azure/key-vault/secrets/about-secrets).
+
+A secret value can be retrieved with the `azureKeyVault` template function, for
+example:
+
+```
+exampleSecret = {{ azureKeyVault "my-secret-name" }}
+```
+
+The Azure Key Vault URL can be specified in the chezmoi's config file with
+`azureKeyVault.vaultUrl.
+
+Ensure [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) is installed and
+[log in](https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#azureCLI) with
+a user having the right key vault RBAC permissions.
```

Comment:
Am I understanding correctly that the Azure CLI is only required for session authentication and that chezmoi will use that session authentication?

I’m not sure how long Azure CLI sessions remain active, so this might be something worth pointing out.

Reply:
Azure CLI uses OAuth 2.0 and indeed only required for authentication. 
That's the easiest way and most common way to do it.

Alternative ways are also possible with the `DefaultAzureCredential` method which chains all authentication types available.

For more info:

https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure

Reply:
OK. If that‘s the way that it works, that’s the way that it works. It would be nice if there was a way to have the Go SDK prompt for authentication or even open an oauth2 web callback.